mixers/GenericAudioMixer.cpp

```diff
@@ -205,8 +212,9 @@ namespace videocore {
                     const auto hash = std::hash<std::shared_ptr<ISource>>()(lSource);
                     
                     auto it = m_lastSampleTime.find(hash);
-                    
-                    if(it != m_lastSampleTime.end() && (mixTime - it->second) < std::chrono::microseconds(int64_t(m_frameDuration * 0.25e6f))) {
+
+                    // when time pass (mixTime - it->second) will be negative number, use abs to compare
+                    if(it != m_lastSampleTime.end() && abs(mixTime - it->second) < std::chrono::microseconds(int64_t(m_frameDuration * 0.25e6f))) {
```

Comment:
@guoqingzhu87 
話したと思いますが、こちらの条件式はまずいことになりますか？
```suggestion
                    auto diff = mixTime - it->second;
                    if(it != m_lastSampleTime.end() && (diff < std::chrono::microseconds(int64_t(m_frameDuration * 0.25e6f)) || diff < 0) {
```